doc/man3/OSSL_CMP_ITAV_set0.pod

```diff
@@ -43,21 +49,32 @@ the I<itav> as generic B<ASN1_TYPE> pointer.
 OSSL_CMP_ITAV_push0_stack_item() pushes I<itav> to the stack pointed to
 by I<*itav_sk_p>. It creates a new stack if I<*itav_sk_p> points to NULL.
 
+OSSL_CMP_ITAV_new0_certProfile() creates a new B<OSSL_CMP_ITAV> structure
+of type B<certProfile> that includes the optionally given list of profile names.
+On success, ownership of the list is with the new B<OSSL_CMP_ITAV> structure.
+
+OSSL_CMP_ITAV_get0_certProfile() on success assigns to I<*out>
+an internal pointer to the
+list of certificate profile names contained in the infoValue field of I<itav>.
+The pointer may be NULL if no profile name is included.
+It is an error if the infoType of I<itav> is not B<certProfile>.
```

Comment:
Is there somewhere in the docs where you define what the types actually are?
When you say certProfile, this seems like  an internal detail (i.e. this is an opaque structure). Do you list the OID's somewhere?

Reply:
For most part, except for applicability of `OSSL_CMP_ITAV_get0_certProfile()`,  
users of the two new functions can consider the type components of `OSSL_CMP_ITAV` (where ITAV is short for InfoTypeAndValue) as internal detail.

These types are represented by OIDs, and those needed for CMPv2+ have been introduced by #15778 in `include//openssl/obj_mac.h`.
The specific `id-it-certProfile` has been defined by https://datatracker.ietf.org/doc/html/rfc9480#name-new-section-5113-certprofil as `OBJECT IDENTIFIER ::= {id-it 21}`.

I've added references in the `NOTES` section as follows:
```
OIDs to use as types in B<OSSL_CMP_ITAV> can be found at
L<https://datatracker.ietf.org/doc/html/rfc9480#section-4.2.2>.
The respective OpenSSL NIDs, such as B<NID_id_it_certProfile>,
are defined in the F<< <openssl/obj_mac.h> >> header file.
```